demos/python_demos/gesture_recognition_demo/gesture_recognition_demo.py

```diff
@@ -66,11 +65,14 @@ def build_argparser():
                       help='Required. Path to a video file or a device node of a web-camera.',
                       required=True, type=str)
     args.add_argument('-c', '--class_map',
-                      help='Required. Path to a file with ASL classes.',
+                      help='Required. Path to a file with gesture classes.',
                       required=True, type=str)
     args.add_argument('-s', '--samples_dir',
                       help='Optional. Path to a directory with video samples of gestures.',
                       default=None, type=str)
+    args.add_argument('-t', '--action_threshold',
+                      help='Optional. Threshold for the predicted score of an action.',
+                      default=None, type=float)
```

Comment:
```suggestion
                      default=0.8, type=float)
```
So you can remove `ACTION_SCORE_THRESHOLD`.

Reply:
Done